tensorzero-internal/src/inference/types/mod.rs

```diff
@@ -2996,4 +2888,58 @@ mod tests {
             _ => panic!("Expected thought block"),
         }
     }
+
+    #[test]
+    fn test_sanitize_input() {
+        assert_eq!(
+            sanitize_raw_request(&[], "my-fake-input".to_string()),
+            "my-fake-input"
+        );
+
+        assert_eq!(
+            sanitize_raw_request(
```

Comment:
should we move this and the sanitize function also to image? don't have a strong opinion really